lib/trino-parquet/src/main/java/io/trino/parquet/writer/valuewriter/UuidValueWriter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.parquet.writer.valuewriter;
+
+import io.airlift.slice.Slice;
+import io.trino.spi.block.Block;
+import io.trino.spi.type.Type;
+import org.apache.parquet.column.values.ValuesWriter;
+import org.apache.parquet.io.api.Binary;
+import org.apache.parquet.schema.PrimitiveType;
+
+import static java.util.Objects.requireNonNull;
+
+public class UuidValueWriter
+        extends PrimitiveValueWriter
+{
+    private final ValuesWriter valuesWriter;
+    private final Type type;
+
+    public UuidValueWriter(ValuesWriter valuesWriter, Type type, PrimitiveType parquetType)
+    {
+        super(parquetType, valuesWriter);
+        this.valuesWriter = requireNonNull(valuesWriter, "valuesWriter is null");
+        this.type = requireNonNull(type, "type is null");
```

Comment:
we don't need this to be parameterized with type, since the type is fixed.
removed the parameter instead (as in some other value writers, eg TimestampTzMillisValueWriter)